src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -145,6 +145,15 @@ public static LocalizableErrorArgument Localize(this MessageID id)
             return new LocalizableErrorArgument(id);
         }
 
+        internal static string RequiredFeature(this MessageID feature)
```

Comment:
`RequiredVersion` is just below this with a similar name (although that could be changed too). For reference, VB does `GetLanguageVersion` (and I'm about to submit a PR that adds `GetFeatureFlag`). I feel like `RequiredFeature` reads better, although I think your example name sounds like a conversion from a feature flag string to a MessageID, which it's the opposite.